zellij-server/src/panes/floating_panes/mod.rs

```diff
@@ -157,7 +157,7 @@ impl FloatingPanes {
 
         // move clients from the previously active pane to the new pane we just inserted
         self.move_clients_between_panes(pane_id, with_pane_id);
-        self.set_pane_frames();
+        let _ = self.set_pane_frames();
```

Comment:
you can just, I believe behavior is the same
```suggestion
        _ = self.set_pane_frames();
```

Reply:
Thanks, I appreciate the comment but I see this as cosmetic